etc/check-bins-for-jemalloc.sh

```diff
@@ -0,0 +1,73 @@
+# This script makes sure that the executables in the target directory
+# are linked to jemalloc. It is suitable to run as part of CI.
+
+#!/bin/bash
+
+dir="`pwd`"
+
+# A list of directories to search for binaries. We're just hoping that `$dir`
+# doesn't contain a space, because that will break the program logic where this
+# string (and others) are interpreted as space-separated lists.
+dirs="$dir/target/debug $dir/target/release"
```

Comment:
can we enter this dir at first, so we don't need to care about the "space" problem?

Reply:
Yeah I can do that.